content/posts/newsletter/2019-10-01.md

```diff
@@ -0,0 +1,124 @@
+
+The igniterealtime community has many news:
+* [Openfire 4.4.2](https://discourse.igniterealtime.org/t/openfire-4-4-2-release/86209): "This release should better support server to server (s2s) connections, fix a few admin console XSS-style issues, and improve client session stability."
+* The Fastpath Service plugin has been released in versions [4.4.4](https://discourse.igniterealtime.org/t/fastpath-service-plugin-4-4-4-released/86155) and [4.4.5](https://discourse.igniterealtime.org/t/fastpath-service-plugin-4-4-5-released/86198), bringing support for managed queued chat requests, such as a support team might use.
+* [Search plugin 1.7.3](https://discourse.igniterealtime.org/t/search-plugin-1-7-3-released/86207): "This update 3 adds protection against CSRF and XSS attacks."
+* [Monitoring Service plugin 1.8.1](https://discourse.igniterealtime.org/t/monitoring-service-plugin-1-8-1-released/86208): "This hotfix update 4 adds protection against XSS attacks on Archiving Settings page."
+
+Jérôme Sautret, from ProcessOne, has announced [ejabberd 19.09](https://blog.process-one.net/ejabberd-19-09/), that brings improved automatic certificate management stack.
+
+### Clients and applications
+
+Profanity, has been released in [version 0.7.0](https://github.com/profanity-im/profanity/releases/tag/0.7.0) after five months of work, brining OMEMO end-to-end encryption, followed by a [0.7.1](https://github.com/profanity-im/profanity/releases/tag/0.7.1) bug fix release.
+
+[Multiple Vulnerabilities have been found in Dino](https://gultsch.de/dino_multiple.html), please update as soon as possible if you are a Dino user.
+
+Converse has been released in versions [5.0.2](https://github.com/conversejs/converse.js/releases/tag/v5.0.2) and [5.0.3](https://github.com/conversejs/converse.js/releases/tag/v5.0.3) fixing security issues among others.
+Converse users may find useful the new [plugin to verify HTTP Requests via XMPP](https://blog.agayon.be/converse_xep_0070.html) from Agayon.
+For developers, there is a [Converse Docker image](https://github.com/conversejs/converse.js-docker) by odajay. 
+
+[Conversations has been released in version 2.5.8](https://github.com/siacs/Conversations/releases/tag/2.5.8).
+
+[Monal 4 with iOS 13 support and dark mode is out](https://monal.im/blog/monal-4-out/). Mac update is expected to be already released for October.
+
+## Services
+
+[Christopher Muclumbus][https://search.jabbercat.org/), a listing and search engine of public XMPP chats, has been updated with a visual redesign, group chat avatars, link to anonymous web chat and logs if available, and software version pie chart.
```

Comment:
I am currently in the middle of yet another last major design overhaul for search.jabbercat.org, which includes a name change.  This isn’t live yet, but will be before the newsletter is merged.

Change the name and domain to ``search.jabber.network`` please.

Reply:
Done, thx

Reply:
```
[search.jabber.network][https://search.jabber.network/), a listing and search engine of public XMPP chat rooms, has had a re-launch. It features a leaner and more accessible design, support for avatars and links to anonymous web chats and logs if available as well as other minor updates and bug fixes.
```